demo/full/scripts/components/Options/VideoAdaptiveSettings.tsx

```diff
@@ -52,6 +54,19 @@ function VideoAdaptiveSettings({
       break;
   }
 
+  let videoResolutionLimitDescMsg;
+  switch (videoResolutionLimit) {
+    case "none":
+      videoResolutionLimitDescMsg = "No limit on the video Representation’s resolution will be automatically applied.";
+      break;
+    case "screen":
+      videoResolutionLimitDescMsg = "The loaded video Representation will be throttled according to the screen’s dimensions.";
+      break;
+    case "videoElement":
+      videoResolutionLimitDescMsg = "The loaded video Representation will be throttled according to the given videoElement’s dimensions.";
+      break;
+  }
```

Comment:
Maybe we should have a column limit to the demo.
We have one for the source but it does not seem to be enabled for the demo code.

Reply:
Apparently there is one but with the options`"ignoreStrings": true` that ignores lines that contains a string 
I'm ok to remove this, but it should be done in a separate PR as there will be probably a lot of changes when correcting all warnings

Reply:
OK I agree